etc/profile-a-l/koi.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for koi
+# Description: Theme scheduling for the KDE Plasma Desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include koi.local
+# Persistent global definitions
+include globals.local
+
+noexec ${HOME}
+noexec /tmp
+
+# Restriction below breaks the program on Arch.
+#include disable-common.inc
+
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+#include disable-shell.inc
+#include disable-x11.inc
+include disable-xdg.inc
+
+apparmor
+caps.drop all
+#ipc-namespace
+machine-id
+
+# Uncomment the restriction below if you don't use Sunset/Sunrise feature.
```

Comment:
Suggestion:

# Add `net none` to your koi.local if you don't use the Sunset/Sunrise feature.